### DISCUSSION TITLE
CNCF & Google Summer of Code 2024

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#1221] [Announcements]
Google has selected 11 CNCF projects to participate in this year's Google Summer of Code! 🎉

| Project Title | Contributor | Assigned Mentor(s) |
| --- | --- | --- |
| Kubernetes Gateway API: Design and develop notification mechanism for ingress2gateway. | Devaansh Kumar (@Devaansh-Kumar) | Lior Lieberman (@LiorLieberman), Mattia Lavacca (@mlavacca) |
| Falco: Upgrading event-generator and automating Falco performance testing | GLVS Kiriti (@GLVSKiriti) | Jason Dellaluce (@jasondellaluce), Aldo Lacuku (@alacuku) |
| CNCF GOSST Collaboration - Enhancing Security Across CNCF Ecosystem | Harshita Sao (@harshitasao) | Nate W. (@nate-double-u ), Dustin Ingram (@di) |
| Building a KubeArmor Extension for Rancher Manager | Krish Aryan (@KrishAryan) | Barun Acharya (@daemon1024), Rudraksh Pareek (@DelusionalOptimist) |
| Add Created Timestamp to Prometheus/OpenMetrics Text Format | Manik Rana (@Maniktherana) | Bartek Plotka (@bwplotka), Daniel Hrabovcak (@TheSpiritXIII) |
| Service Mesh Performance | Shlok Mishra (@Jougan-0) | Lee Calcote (@leecalcote), Mohd Uzair Shaikh (@muzairs15) |
| KubeArmor: Automating Benchmarking Workflow | Shubh Gaur (@sratslla) | Barun Acharya (@daemon1024), Shreyas Mishra (@Shreyas220), Rudraksh Pareek (@DelusionalOptimist) |
| Implementing Coredump for Post-Mortem Debugging in WebAssembly | Siddharth Tewari (@sid-008) | Hung-Ying Tai (@hydai), Shen-Ta Hsieh (@ibmibmibm) |
| Move out of gogo/protobuf to protobuf with vtproto implementation | Tony Li (@tonyli233) | Max Amin (@macxamin), Callum Styan (@cstyan) |
| GSoC 2024 Proposal for CNCF(Open Cluster Management) | Zhe Shen (@z1ens) | Qing Hao (@haoqing0110), Jian Qiu (@qiujian16) |
| Improving WASM Support in Knative Functions | Zoey_Hong (@iam-zoey) | Matthias Wessendorf (@matzew), Aleksander Slominski (@aslom), Luke Kingland (@lkingland) |

Next up is the Community Bonding period (May 1 - 25), so feel free to use this discussion to say hello. The Community Bonding period is intended to help prepare contributors to start contributing to your organization at full speed starting May 27, so mentors, be sure to reach out to your mentees!

Congratulations everyone, this is a strong cohort, and I think it's going to be a fun summer!

**Comment by Devaansh-Kumar** [comment]
Hey @nate-double-u and all other contributors. As the season comes to an end I wanted to tell that I had a fun time with CNCF during my GSoC period. I hope everyone enjoyed contributing as well. Thanks for selecting me for this journey.

Is there a possibilty that CNCF can provide the mentees with some swags/goodies :upside_down_face: :pray:
